common/web/types/src/kpj/kpj-file.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Keyman Developer Project File
+ */
+
+
+// These interfaces match XML read by xml2js. Use `KeymanDeveloperProject`
+// returned by `KPJFileReader.transform()` for all processing.
+
+export interface KPJFile {
+  KeymanDeveloperProject: KPJFileProject;
+}
+
+export interface KPJFileProject {
+  Options?: KPJFileOptions;
+  Files?: KPJFileFiles;
+};
+
+export interface KPJFileOptions {
+  BuildPath?: string;
+  CompilerWarningsAsErrors?: string; // default False
+  WarnDeprecatedCode?: string; // default True
+  CheckFilenameConventions?: string; // default True
+  ProjectType?: 'keyboard' | 'lexicalmodel';
```

Comment:
Are these defaults for comments or TODOs?
And is default ProjectType keyboard?

Reply:
These defaults are just comments